packages/deployments/contracts/contracts/core/connext/facets/BaseConnextFacet.sol

```diff
@@ -135,26 +137,27 @@ contract BaseConnextFacet {
   }
 
   /**
-   * @notice Calculates a transferId
+   * @notice Returns the adopted assets for given canonical information
    */
-  function _calculateTransferId(CallParams memory _params) internal pure returns (bytes32) {
-    return keccak256(abi.encode(_params));
+  function _getRepresentationAsset(bytes32 _canonicalId, uint32 _canonicalDomain) internal view returns (address) {
+    return _getRepresentationAsset(AssetLogic.calculateCanonicalHash(_canonicalId, _canonicalDomain));
```

Comment:
nit: imo it'd be better to require that the caller calculate the canonical hash separately ahead of time (i.e. *only* provide the `_getRepresentationAsset(bytes32 _key)` function below and no convenience functions like this). That way we don't end up doing redundant work.

more of a nit since it's really not that expensive to hash things.. just kinda seems best practice to me